<launch>
    <node name="pub_dt2_bndpts" pkg="sitl_dvrk" type="pub_dt2_bndpts.py" output="screen">
        <param name="calib_dir"        type="str"    value="L2R"/>
        <param name="frame_id"         type="str"    value="ecm_opencv"/>
        <param name="obj_type"         type="str"    value="chicken"/>
        <param name="model_path"       type="str"    value="/home/$(env USER)/catkin_ws/src/sitl_dvrk/scripts/detectron2/Segmentation/Seg_LiverChicken_v3.pth"/>
        <param name="save_path"        type="str"    value="/home/$(env USER)/auto_dissect_traj"/>
        <param name="score_thresh"     type="double" value="0.5"/>
        <param name="num_goals"        type="int"    value="8"/>
        <param name="cnt_area_thr"     type="int"    value="10000"/>
    </node>

    <?ignore
    
    ?>
</launch>